README.md

```diff
@@ -82,6 +82,15 @@ The `pytest-tinybird` plugin creates and sends `report` objects via the [Events
 }
 ```
 
+There are also additional optional values that can be set for multi-repository and multi-workflow setups (e.g., in GitHub Actions):
```

Comment:
What about mentioning which env variables need to be set? All other env variables used are from [gitlab predefined ones](https://docs.gitlab.com/ci/variables/predefined_variables/#predefined-variables) not to be manually defined. Maybe worth to add that this two new variables are custom, not from GitLab